certgen/certProcessor/src/main/java/org/incredible/certProcessor/views/HTMLZipProcessor.java

```diff
@@ -0,0 +1,102 @@
+package org.incredible.certProcessor.views;
+
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.net.URL;
+import java.util.Enumeration;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+public class HTMLZipProcessor extends HTMLTemplateProvider {
```

Comment:
Rename this like others - HTMLTemplateZip. And can it not derive from HTMLTemplateFile to reuse getTemplateContent()